----------------------------------------------------
Checking source files for common errors in following
directories:
deployments src tests perf_tests PoC_code
----------------------------------------------------

Using base prefix '/usr'
New python executable in /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/repositories/f8a-hpf-insights/venv/bin/python3
Not overwriting existing python script /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/repositories/f8a-hpf-insights/venv/bin/python (you must use /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/repositories/f8a-hpf-insights/venv/bin/python3)
Installing setuptools, pip, wheel...done.
Running virtualenv with interpreter /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/venv/bin/python3
Requirement already satisfied: pyflakes in ./venv/lib/python3.6/site-packages (2.0.0)
deployments/__init__.py
    Pass
src/data_store/__init__.py
    Pass
src/data_store/abstract_data_store.py
    Pass
src/data_store/local_data_store.py
    Pass
src/data_store/s3_data_store.py
    Pass
src/__init__.py
    Pass
src/flask_endpoint.py
    Pass
src/scoring/__init__.py
    Pass
src/scoring/hpf_scoring.py
src/scoring/hpf_scoring.py:10: redefinition of unused 'os' from line 5
src/scoring/hpf_scoring.py:12: 'collections.defaultdict' imported but unused
src/scoring/hpf_scoring.py:14: 'src.config.c' imported but unused
src/scoring/hpf_scoring.py:14: 'src.config.c_c' imported but unused
src/scoring/hpf_scoring.py:14: 'src.config.d_c' imported but unused
    Fail
src/training/__init__.py
    Pass
tests/__init__.py
    Pass
tests/unit_tests/test_scoring_hpf_scoring.py
tests/unit_tests/test_scoring_hpf_scoring.py:5: 'tensorflow as tf' imported but unused
    Fail
tests/unit_tests/__init__.py
    Pass
tests/unit_tests/test_local_datastore.py
    Pass
perf_tests/__init__.py
    Pass
perf_tests/locust_tests.py
perf_tests/locust_tests.py:4: 'locust.events' imported but unused
perf_tests/locust_tests.py:4: 'locust.web' imported but unused
perf_tests/locust_tests.py:99: local variable 'response' is assigned to but never used
    Fail
PoC_code/__init__.py
    Pass
3 source files out of 17 files needs to be checked and fixed
